extension/data.ts

```diff
@@ -86,6 +86,17 @@ interface DeinflectionRuleGroup {
   rules: DeinflectionRule[];
 }
 
+// prettier-ignore
+const CHAR_CODES_BELOW_0x3000_USED_IN_WORDS_IN_DICTIONARY = [
+  '×', 'α', '○', '◎', '△', '※', '-', '°', 'R',
+  'N', 'A', '♀', 'Α', 'Β', 'Γ', 'Δ', 'Ε', 'Ζ',
+  'Η', 'Θ', 'Ι', 'Κ', 'Λ', 'Μ', 'Ν', 'Ξ', 'Ο',
+  'Π', 'Ρ', 'Σ', 'Τ', 'Υ', 'Φ', 'Χ', 'Ψ', 'Ω',
+  'β', 'γ', 'δ', 'ε', 'ζ', 'η', 'θ', 'ι', 'κ',
+  'λ', 'μ', 'ν', 'ξ', 'ο', 'π', 'ρ', 'σ', 'τ',
+  'υ', 'φ', 'χ', 'ψ', 'ω'
+].map(char => char.charCodeAt(0));
```

Comment:
Let me know if I should change this code so that it has the character codes already computed rather than calling Array.prototype.map here.

I chose to leave it like this so that the characters are human-readable, but I can also change it to just show the character codes if that's preferred.

Reply:
This seems reasonable to me.